pkg/test/scenario/example_noop_test.go

```diff
@@ -22,10 +23,7 @@ var noopScenario Scenario = Scenario{
 		},
 	},
 	Spec: model.Spec{
-		Engine: model.EngineNoop,
-		Wasm: model.JobSpecWasm{
-			EntryPoint: "_start",
-		},
+		EngineSpec: model.EngineSpec{Type: model.EngineNoop},
```

Comment:
I wasn't sure to handle this specific conversion to the new `EngineSpec`. But my sense was that this test expects a noop engine and I have attempted to keep the same behavior.